.ai/spec/how/components.md

```
@@ -0,0 +1,195 @@
+| `AttachmentLabel.tsx` | Inline | Attachment display label |
+| `AttachEventsModal.tsx` | Modal | Event selection for attachment |
+| `AttachLogModal.tsx` | Modal | Log selection for attachment |
+| `AttachmentsSizeAlert.tsx` | Alert | Warning for large attachments |
+| `ErrorBoundary.tsx` | Utility | React error boundary |
+| `Modal.tsx` | Utility | Reusable modal wrapper |
+
+## Data Flow
+
+### Component tree
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)